samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Attribute.Equals/VB/equals.vb

```diff
@@ -1,161 +1,135 @@
+                    GetType(ArgumentUsageAttribute))
                  
-                ' Create an instance of the argument usage attribute on strList.
-                Dim listUsageAttr As ArgumentUsageAttribute = _
-                    Attribute.GetCustomAttribute(pInfoArray(1), _
-                        GetType(ArgumentUsageAttribute))
+            ' Create an instance of the argument usage attribute on strList.
+            Dim listUsageAttr As ArgumentUsageAttribute = _
+                Attribute.GetCustomAttribute(pInfoArray(1), _
+                    GetType(ArgumentUsageAttribute))
                  
-                ' Create an instance of the argument ID attribute on strArray.
-                Dim arrayIDAttr1 As ArgumentIDAttribute = _
-                    Attribute.GetCustomAttribute(pInfoArray(0), _
-                        GetType(ArgumentIDAttribute))
+            ' Create an instance of the argument ID attribute on strArray.
+            Dim arrayIDAttr1 As ArgumentIDAttribute = _
+                Attribute.GetCustomAttribute(pInfoArray(0), _
+                    GetType(ArgumentIDAttribute))
                  
-                ' Create another instance of the argument ID attribute on strArray.
-                Dim arrayIDAttr2 As ArgumentIDAttribute = _
-                    Attribute.GetCustomAttribute(pInfoArray(0), _
-                        GetType(ArgumentIDAttribute))
-                
-                ' Create an instance of the argument ID attribute on strList.
-                Dim listIDAttr As ArgumentIDAttribute = _
-                    Attribute.GetCustomAttribute(pInfoArray(1), _
-                        GetType(ArgumentIDAttribute))
+            ' Create another instance of the argument ID attribute on strArray.
+            Dim arrayIDAttr2 As ArgumentIDAttribute = _
+                Attribute.GetCustomAttribute(pInfoArray(0), _
+                    GetType(ArgumentIDAttribute))
+              
+            ' Create an instance of the argument ID attribute on strList.
+            Dim listIDAttr As ArgumentIDAttribute = _
+                Attribute.GetCustomAttribute(pInfoArray(1), _
+                    GetType(ArgumentIDAttribute))
                  
-                ' Compare various pairs of attributes for equality.
-                Console.WriteLine(vbCrLf & "Compare a usage attribute instance " & _
-                    "to another instance of the same attribute:")
-                Console.WriteLine("   ""{0}"" = " & vbCrLf & "   ""{1}"" ? {2}", _
-                    arrayUsageAttr1.ToString(), arrayUsageAttr2.ToString(), _
-                    arrayUsageAttr1.Equals(arrayUsageAttr2))
+            ' Compare various pairs of attributes for equality.
+            Console.WriteLine(vbCrLf & "Compare a usage attribute instance " & _
+                "to another instance of the same attribute:")
+            Console.WriteLine("   ""{0}"" = " & vbCrLf & "   ""{1}"" ? {2}", _
```

Comment:
Suggestion: Do we want to replace the {0}, {1}, etc. with string interpolation in this example?

Reply:
Yes, @BillWagner, we definitely want to use interpolated strings. Thanks for noticing that the code didn't. I've changed the examples.